Trying to cleanup the start script

jofemodo opened the pull request:
Remove package install, as it's done by setup script. Including jackd realtime configuration ;-)

It's a first try. If it fails "too much", i will debug the script running by hand from console and will come back with a "working" version.

What about having writing permissions? Or do you prefer i fork the repo and change the nightly scripts in os.zynthian.org?

jofemodo committed:
Trying to cleanup the start script

Remove package install, as it's done by setup script. Including jackd realtime configuration ;-)

guysoft commented:
Note you removed

```
DEBIAN_FRONTEND=noninteractive apt-get install -y jackd

# Enable realtime permissions (taken from /etc/security/limits.d/audio.conf.disabled )
dpkg-reconfigure -p high jackd
```

Make sure you put it in zynthian-sys or jack would have latency issues.

jofemodo commented:
Of course ... i added the needed stuff in zynthian-sys ;-)
Regarding this:

`dpkg-reconfigure -p high jackd`

Doesn't work as you could think ... The "-p" flag doesn't set the priority of jackd to "high" as we expected. In fact, the priority of jack has been "low" all this time in the nightly build images!! I just fixed this yesterday and now it should be working OK.

There are some pages that says that this line will do the trick, but not!! Check the "dpkg-reconfigure" manpage and you will understand ... ;-)
```

 -pvalue, --priority=value
           Specify the minimum priority of question that will be displayed.  dpkg-reconfigure
           normally shows low priority questions no matter what your default priority is. See
           debconf(7) for a list.

```

Cheers!
